packages/drafts-realm/app-card.gts

```diff
@@ -290,56 +154,56 @@ class Isolated extends Component<typeof AppCard> {
-                      format='data'
-                      fieldType=undefined
-                      fieldName=undefined
-                    }}
-                    data-test-cards-grid-item={{card.id}}
-                    {{! In order to support scrolling cards into view
+            <ul class='cards-grid' data-test-cards-grid-cards>
+              {{! use "key" to keep the list stable between refreshes }}
+              {{#each this.instances key='id' as |card|}}
+                <li
+                  {{@context.cardComponentModifier
+                    card=card
+                    format='data'
+                    fieldType=undefined
+                    fieldName=undefined
+                  }}
+                  data-test-cards-grid-item={{card.id}}
+                  {{! In order to support scrolling cards into view
             we use a selector that is not pruned out in production builds }}
-                    data-cards-grid-item={{card.id}}
-                  >
-                    <div class='grid-card'>
-                      <div
-                        class='grid-thumbnail'
-                        style={{cssUrl 'background-image' card.thumbnailURL}}
-                      >
-                        {{#unless card.thumbnailURL}}
-                          <div
-                            class='grid-thumbnail-text'
-                            data-test-cards-grid-item-thumbnail-text
-                          >{{cardTypeDisplayName card}}</div>
-                        {{/unless}}
-                      </div>
-                      <h3
-                        class='grid-title'
-                        data-test-cards-grid-item-title
-                      >{{card.title}}</h3>
-                      <h4
-                        class='grid-display-name'
-                        data-test-cards-grid-item-display-name
-                      >{{cardTypeDisplayName card}}</h4>
+                  data-cards-grid-item={{card.id}}
+                >
+                  <div class='grid-card'>
+                    <div
+                      class='grid-thumbnail'
+                      style={{cssUrl 'background-image' card.thumbnailURL}}
+                    >
+                      {{#unless card.thumbnailURL}}
+                        <div
+                          class='grid-thumbnail-text'
+                          data-test-cards-grid-item-thumbnail-text
+                        >{{cardTypeDisplayName card}}</div>
+                      {{/unless}}
                     </div>
-                  </li>
+                    <h3
+                      class='grid-title'
+                      data-test-cards-grid-item-title
+                    >{{card.title}}</h3>
+                    <h4
+                      class='grid-display-name'
+                      data-test-cards-grid-item-display-name
+                    >{{cardTypeDisplayName card}}</h4>
+                  </div>
+                </li>
```

Comment:
Should this be using the embedded format of the card?

Reply:
yes. This seems to be copied over from `cards-grid` in base. the card I worked on is not using this view yet. but it's worth refactoring so that when we use a grid view on the new index card for example, the embedded version can be used.